src/app/compatibility/weekly_contributions.tsx

```diff
@@ -39,7 +39,7 @@ export function WeeklyContributions({ data }: { data: DataPoint[] }) {
       series={[{ name: "Commits", color: "ruffle-orange" }]}
       tooltipProps={{
         content: ({ label, payload }) => (
-          <ChartTooltip label={label} payload={payload} />
+          <ChartTooltip label={String(label || "")} payload={payload} />
```

Comment:
Why not change `label: string;` in `ChartTooltipProps` to `label?: string;` and change `if (!payload) return null;` to `if (!payload || !label) return null;` in `ChartTooltip`.

Reply:
as you noted on Discord, since we use string labels anyway, I don't see the point...?